pymc_marketing/mmm/delayed_saturated_mmm.py

```diff
@@ -142,6 +144,112 @@ def _save_input_params(self, idata) -> None:
         idata.attrs["validate_data"] = json.dumps(self.validate_data)
         idata.attrs["yearly_seasonality"] = json.dumps(self.yearly_seasonality)
 
+    def _get_distribution(self, dist: Dict) -> Callable:
```

Comment:
Should this be in the base class?

Reply:
mmmm.... Yes, it could be, what would be the difference if we move it? 🧐

Reply:
If anyone subclasses the base class they have access to the same method.